webrtc-stats.html

```diff
@@ -1745,6 +1748,49 @@ <h2>
                   [= map/exist | present =].
                 </p>
               </dd>
+              <dt>
+                <dfn>totalCorruptionProbability</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio. Represents the cumulative sum of all corruption
+                  probability measurements that have been made for this SSRC, see {{corruptionMeasurements}}
+                  regarding when this attribute SHOULD be [= map/exist | present =].
+                </p>
+              </dd>
+              <dt>
+                <dfn>totalSquaredCorruptionProbability</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio. Represents the cumulative sum of all corruption
+                  probability measurements squared that have been made for this SSRC, see
+                  {{corruptionMeasurements}} regarding when this attribute SHOULD be [= map/exist | present =].
+                </p>
+              </dd>
+              <dt>
+                <dfn>corruptionMeasurements</dfn> of type <span class=
+                "idlMemberType">unsigned long long</span>
+              </dt>
+              <dd>
+                <p>
+                  MUST NOT [= map/exist =] for audio. When the user agent is able to make a corruption
+                  probability measurement, this counter is incremented for each such measurement and
+                  {{totalCorruptionProbability}} and {{totalSquaredCorruptionProbability}} are aggregated
+                  with this measurement and measurement squared respectively.
+                </p>
+                <p class="note">
```

Comment:
From editor's meeting:
- @youennf and @jan-ivar prefers to convert from a note to more normative language. Can we say MUST and point to this header extension, rather than saying "anything"?
- If we can't say exactly what the algorithm is, can we at least define what a acceptable algorithm behaves?
- An adopted IETF draft is needed to reference a link that isn't /experiments/, this would be a blocker for candidate recommendation but not a blocker for merging this PR, can we add a note that says something like this?